bedrock/mozorg/templates/mozorg/contact/contact-landing.html

```diff
@@ -0,0 +1,66 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "mozorg/contact/contact-base.html" %}
+
+{% block page_title %}{{ _('Contacts, Spaces and Communities') }} — {{ _('Contact us') }}{% endblock %}
+
+{% block contact_entry %}
+  <div id="entry-container">
+    <section class="entry contact-us" id="contact" data-tab="contact">
+      <header>
+        <h2>{{ _('Contacting Mozilla')}}</h2>
+      </header>
+
+      <ul>
+        <li><a href="https://support.mozilla.org/">{{ _('I’m having problems with using Firefox')}}</a></li>
+        <li><a href="https://sendto.mozilla.org/page/contribute/givenow-seq?preset=2&ref=Donation_Form&utm_campaign=Donation_Form&utm_source=mozilla.org&utm_medium=referral&utm_content=contact_us_page">{{ _('I want to donate to Mozilla')}}</a></li>
+        <li><a href="https://www.mozilla.org/foundation/trademarks/policy/">{{ _('I have questions about copyright trademarks and Mozilla')}}</a></li>
+        <li><a href="mailto:research_requests@mozilla.com">{{ _('I need information for an academic paper')}}</a></li>
+        <li><a href="https://wiki.mozilla.org/People:MozSpaces_Guidelines">{{ _('I want to hold an event in a Mozilla space')}}</a></li>
+        <li><a href="https://bugzilla.mozilla.org/form.dev-engagement-event">{{ _('I want Mozilla to sponsor my event')}}</a></li>
+        <li><a href="https://www.mozilla.org/styleguide/identity/mozilla/branding/">{{ _('I have a question about using Mozilla logos')}}</a></li>
+        <li><a href="#press_enquiry" id="press_enquiry_toggle">{{ _('I’m a member of the Press and have a question for Mozilla') }}</a></li>
+      </ul>
+
+      <p>
+      {% trans link_sumo='https://support.mozilla.org' %}
+        To reach Mozilla anywhere in the world, check out the locations above. For help
+        with any of our products, including Firefox for desktop and Android, Firefox OS
+        and Thunderbird, please <a href="{{ link_sumo }}">visit our support site</a>.
+      {% endtrans %}
+      </p>
+
+      <div class="contact-online">
+        <h4>{{ _('Find us online') }}</h4>
+        <ul>
+          <li><a href="https://facebook.com/mozilla" hreflang="en-US">{{ _('Facebook') }}</a></li>
+          <li><a href="https://twitter.com/mozilla" hreflang="en-US">{{ _('Twitter') }}</a></li>
+          <li><a href="/about/forums/">{{ _('Forums &amp; e-mail lists') }}</a></li>
```

Comment:
Yeah, good point. This was all copied from preexisting code but, no reason not to improve it ;)